test/fake/fake-currency-plugin.js

```diff
@@ -314,6 +314,11 @@ class FakeCurrencyEngine {
   saveTx(transaction: EdgeTransaction): Promise<void> {
     return Promise.resolve()
   }
+
+  // Accelerating:
+  accelerate(transaction: EdgeTransaction): Promise<EdgeTransaction | null> {
```

Comment:
We should upgrade this repo to use async more consistently, but we started coding in here before that was an option. I would rather see `async accelerate() (...) {return null}` going forward.